Bottom: 745ee9825c306767844496be2a8533cdf7c9c170
Top:    0a7024b7ae9a9367a5076b9de237cd5b3626804c
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-18 16:07:12 +0800

Refresh of tun-reserve-extra-headroom

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 711e67b..674a8e2 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1272,7 +1272,7 @@ static struct sk_buff *tun_build_skb(struct tun_struct *tun,
 	struct page_frag *alloc_frag = &current->task_frag;
 	struct sk_buff *skb;
 	struct bpf_prog *xdp_prog;
-	int buflen = SKB_DATA_ALIGN(sizeof(struct skb_shared_info);
+	int buflen = SKB_DATA_ALIGN(sizeof(struct skb_shared_info));
 	unsigned int delta = 0;
 	char *buf;
 	size_t copied;
